tools/py_common/btree_format.py

```diff
@@ -178,26 +194,92 @@ def parse(b: binary_data.BinaryFile, disagg = False) -> 'BlockHeader':
         '''
         # WT_BLOCK_HEADER in block.h (12 bytes)
```

Comment:
disagg is not used is not used in this function

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/12905/commits/a28bfbfc2d95be0631e78f3c96aedeb95a3c36af